[dbus] Build error

Package: dbus:x64-linux -> 1.15.8

**Host Environment**

- Host: x64-linux
- Compiler: GNU 10.2.1
-    vcpkg-tool version: 2023-09-15-ac02a9f660977426b8ec6392919fbb1d51b10998
    vcpkg-scripts version: cc97b45 2023-10-06 (16 hours ago)

**To Reproduce**
*(issue_body is too long, please find it in attachment)*
[vcpkg-sdl2-failure.md](https://github.com/microsoft/vcpkg/files/12838217/vcpkg-sdl2-failure.md)

**Long story short**
Start new docker container with following command:

```
docker run -ti quay.io/pypa/manylinux2014_x86_64 bash
```

NOTE: command above uses latest version of manylinux_2014_x86_64, consider using direct
image ID, which is 43f32dc5292e

Inside docker container run following commands:

```bash
yum install -y zip wget
git clone --depth=1 https://github.com/microsoft/vcpkg /vcpkg
cd /vcpkg
./bootstrap-vcpkg.sh -disableMetrics
./vcpkg install sdl2 # BOOM
```

~~~
-- Configuring done (13.2s)
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
X11_Xdmcp_LIB (ADVANCED)
    linked by target "dbus-launch" in directory /vcpkg/buildtrees/dbus/src/bus-1.15.8-4cdd406cc7.clean/tools

-- Generating done (0.0s)
~~~
Hm, this port seems to ask for some X libs. But it shouldn't. The feature control history points at me, #27425 :facepalm:

--- a/ports/dbus/portfile.cmake
+++ b/ports/dbus/portfile.cmake
@@ -13,7 +13,7 @@ vcpkg_from_gitlab(
         getpeereid.patch # missing check from configure.ac
 ) 
 
-vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+vcpkg_check_features(OUT_FEATURE_OPTIONS options
     FEATURES
         x11     DBUS_BUILD_X11
         x11     CMAKE_REQUIRE_FIND_PACKAGE_X11
@@ -36,6 +36,7 @@ vcpkg_cmake_configure(
         "-DCMAKE_INSTALL_SYSCONFDIR=${CURRENT_PACKAGES_DIR}/etc/${PORT}"
         "-DWITH_SYSTEMD_SYSTEMUNITDIR=lib/systemd/system"
         "-DWITH_SYSTEMD_USERUNITDIR=lib/systemd/user"
+        ${options}
     OPTIONS_RELEASE
         -DDBUS_DISABLE_ASSERT=OFF
         -DDBUS_ENABLE_STATS=OFF

--- a/ports/dbus/vcpkg.json
+++ b/ports/dbus/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "dbus",
   "version": "1.15.8",
+  "port-version": 1,
   "description": "D-Bus specification and reference implementation, including libdbus and dbus-daemon",
   "homepage": "https://gitlab.freedesktop.org/dbus/dbus",
   "license": "AFL-2.1 OR GPL-2.0-or-later",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2094,7 +2094,7 @@
     },
     "dbus": {
       "baseline": "1.15.8",
-      "port-version": 0
+      "port-version": 1
     },
     "dcmtk": {
       "baseline": "3.6.7",

--- a/versions/d-/dbus.json
+++ b/versions/d-/dbus.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "9104cc6d890ace8c14067e7cbf29c39f7a93b937",
+      "version": "1.15.8",
+      "port-version": 1
+    },
+    {
       "git-tree": "a49c03bc95822c2f376e3f3758ac3087a428ddec",
       "version": "1.15.8",
       "port-version": 0

